src/components/heading/heading.test.js

```diff
@@ -152,6 +152,19 @@ context("Testing Heading component", () => {
       }
     );
 
+    describe("when headingType prop is provided", () => {
+      it.each(["h1", "h2", "h3", "h4", "h5", "h6"])(
+        "should check HTML section heading element is correct when headingType is %s",
+        (headingTypes) => {
```

Comment:
```suggestion
        (headingType) => {
```
this argument represents a single heading type so the singular name is less confusing (`headingType={headingTypes}` on the line below makes my point I feel!)